apps/bfd-pipeline/bfd-pipeline-app/src/main/java/gov/cms/bfd/pipeline/app/AppConfiguration.java

```diff
@@ -616,9 +636,12 @@ static CcwRifLoadOptions loadCcwRifLoadOptions(ConfigLoader config, LoadAppOptio
     } else {
       allowedRifFileType = Optional.empty();
     }
-
-    LayeredConfiguration.ensureAwsCredentialsConfiguredCorrectly();
-    ExtractionOptions extractionOptions = new ExtractionOptions(s3BucketName, allowedRifFileType);
+    final S3ClientConfig s3ClientConfig = loadS3ServiceConfig(config);
+    if (s3ClientConfig.isCredentialCheckUseful()) {
+      LayeredConfiguration.ensureAwsCredentialsConfiguredCorrectly();
```

Comment:
Nice; I think this will remove the last remnant of needing our AWS credentials to run the tests.